worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/BlocksHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.fastasyncworldedit.core.queue.implementation.blocks;
+
+import com.sk89q.worldedit.world.biome.BiomeType;
+
+public class BlocksHelper {
```

Comment:
I feel like a better name for this would be something like `ChunkSectionedChunk`. `BlocksHelper` sounds like it should be a utility class, rather than actually containing data and being a non-final class

Reply:
Its been renamed with the latest commit.